Source/Libraries/CorruptCore/Extensions/ByteArrayComparer.cs

```diff
@@ -1,11 +1,17 @@
 namespace RTCV.CorruptCore.Extensions
 {
+    using System;
     using System.Collections.Generic;
 
     public class ByteArrayComparer : IEqualityComparer<byte[]>
     {
         public bool Equals(byte[] a, byte[] b)
         {
+            if (a == null || b == null)
```

Comment:
Any idea if there's a specification on if we should be returning true if they're both null? Obviously you shouldn't be comparing the two nulls, but I feel like this is something that has to be defined by the standard as having correct behavior.

Reply:
I don't know about any specification. It seems logical to me that `null == null`, so I'm fine added that here and in the other comparator. That seems more correct.

Reply:
From a bit of research, for both reference and value checks, the framework (Object.Equals and Object.ReferenceEquals) both return true if fed null and null. That's pretty good justification of the intent behind the framework devs to me.

Reply:
Done